### ISSUE TITLE
Bug: lack of asterisk * options in class Hotstring

### ISSUE THREAD

**Issue by zzsimplezz** [spyoungtech/ahk#218]
In AHK, you can use asterisk `*` for trigger hotstring without ending character.

Currently the regex assert in `class Hotstring` lacks it.

the regex _validate method in class Hotstring: 
```ahk
(\?|C|C1|K\d+|O|P\d+|S[IPE]|T|Z)+
```
it should add "*" to work: 
```ahk
(\?|C|C1|K\d+|O|P\d+|S[IPE]|T|Z|\*)+
```
thank you for your lib. 😄

**Comment by spyoungtech** [comment]
Yep, that seems like an oversight. Will add the fix in an upcoming release.

**Comment by spyoungtech** [comment]
This has been merged and will be released in the upcoming 1.2.0 release.

**Comment by spyoungtech** [comment]
This has been released in 1.2.0 🎉
